src/core/Directus/Config/Schema/Schema.php

```diff
@@ -38,6 +38,7 @@ public static function get() {
                 new Value('engine', Types::STRING, 'InnoDB'),
                 new Value('charset', Types::STRING, 'utf8mb4'),
                 new Value('socket', Types::STRING, ''),
+                new Value('driver_options', Types::ARRAY, []),
```

Comment:
As this didn't exist before, please use "?" to completely remove the value from the final object if it's not set. I'm not sure if make [] the default value can cause problems.

Reply:
done